docker-compose/setenv.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+if [ $(uname -m) == "x86_64" ] | [ $(uname -m) == "x86_64" ]; then
+    export ARCH_MEMSVC="fabric-membersrvc:x86_64-0.6.0-preview"
+    export ARCH_PEER="fabric-peer:x86_64-0.6.0-preview"
+elif [ $(uname -m) == "s390x" ]; then
+    export ARCH_MEMSVC="fabric-membersrvc:s390x-0.6.0-preview"
```

Comment:
If we pull images from https://hub.docker.com/u/hyperledger/ docker hub account, then change above lines to "fabric-membersrvc-s390x" and "fabric-peer-s390x" for s390x ARCH and "fabric-membersrvc-ppc64le" and "fabric-peer-ppc64le" for ppc64le ARCH..

Note: We yet to push peer and membersrvc images of s390x and ppc64le to hyperledger docker hub account.. right now, it's hanging with empty list..